### ISSUE TITLE
Keyfile creating

### ISSUE THREAD

**Issue by harryongituser** [merdok/homebridge-webos3#38]
Thanks for the plugin. However, I have an issue saving the permission token. In the log, it said error: ENOENT: no such file or directory , open ‘/home/pi/Homebridge/lgtvKeyFile’.
I have set the raspberry pi permission right (don’t know set it right or wrong), still can’t store permission token. Can any one help? Thanks.

**Comment by merdok** [comment]
Hi,

either the directory which you specified doesn't exist or you don't have the correct permissions set.

**Comment by harryongituser** [comment]
What permissions required to be set? I have set with sudo chmod a+w the that directory 
Don’t know if that is right, can you teach me to set what permission .thank you very much.

**Comment by merdok** [comment]
0755, and of course the homebridge (pi) user needs to have write permission to that location.

**Comment by harryongituser** [comment]
Sorry , you mean sudo chmod 755 to that location ?

**Comment by merdok** [comment]
Yes, but the hombridge user (whatever you use for that) also needs to be have write permission to that location. Use the default location if you still have issues since then the location will be created automatically.

**Comment by harryongituser** [comment]
But when I change the permission in raspberry pi the directory with “/.homebridge” said doesn’t exist.
By the way,if the location set to other than the one in your config example doesn’t automatic create the lgtvKeyFile?

**Comment by merdok** [comment]
My config is just an example, you can use whatever directory you want **but** it needs to exist!
You can remove this property completely, then it will be handled for you

**Comment by harryongituser** [comment]
Sorry, still not quite understand. Do you mean I leave the key location config option blank tot automatically create the files to store token ? Thanks a lot

**Comment by merdok** [comment]
Yes, just leave it blank if you don't know what you doing.
